privacyidea/static/components/token/views/token.enrolled.paper.html

```diff
@@ -56,10 +56,10 @@
                 </div>
             </div>
         </uib-accordion>
+        <button class="btn-default btn"
```

Comment:
Why are you moving the button to print in the closed accordeon?
This way users will not see, that they can print the list.

If there is no necessary reason, please move it back.

Reply:
The button is outside the accordion and still visible if the accordion is closed.
I just moved the button inside the `div` with the attribute `ng-show="otp_rows` to disable showing the accordion and button if there are no TAN values.